src/libraries/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -68,7 +68,7 @@ private Func<
             X509Chain,
             SslPolicyErrors,
             bool>? _serverCertificateValidationCallback;
-        private bool _checkCertificateRevocationList;
+        private bool _checkCertificateRevocationList = true;
```

Comment:
Should this be `true`, or should it inspect the appcontext-overridable default mode value?

Reply:
I can't see from the PR, so: Ensure you have tests where this gets forced to false, instead of only tests that assumed it was false.  Ideally tests where the cert is revoked (or the revocation responder is set to explode, or time out, or whatever), so you can tell it worked.